src/components/index.js

```diff
@@ -0,0 +1 @@
+export {StatueListContainer} from './StatueListContainer'
```

Comment:
I've never seen this before, what is it for?

Reply:
It is a way that you can centralise all your exports.
 Having named exports enables you to do this and it makes it easier to manage all your exports.